examples/go-kafkacat/go-kafkacat.go

```diff
@@ -37,6 +42,89 @@ var (
 	sigs         chan os.Signal
 )
 
+var (
+	oauthbearerConfigRegex          = regexp.MustCompile("^(\\s*(\\w+)\\s*=\\s*(\\w+))+\\s*$")
```

Comment:
Add a comment of what format it is trying to match.

Reply:
👍